### DISCUSSION TITLE
Querying using tagged template literals: edgeql`select Movie { title }`

### DISCUSSION THREAD

**Discussion by infogulch** [geldata/gel#4051] [Ideas]
Some GraphQL libraries use a neat feature of js syntax called [tagged template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) that enable embedding parsed queries into js. See:

https://pavsaund.com/post/2020-10-26-tagged-template-literals-how-the-gql-query-syntax-works/

https://github.com/apollographql/graphql-tag

With some preprocessing or [other shenanigans](https://github.com/ghoullier/awesome-template-literal-types) you might even be able to emit types for them.

Have you considered something like this?

**Comment by haikyuu** [comment]
There is an open PR https://github.com/edgedb/edgedb-js/pull/381

P.S: You might get faster responses in Discord

**Reply by infogulch** [reply]
Thanks, I'm not sure why I didn't find that when I searched.

**Reply by infogulch** [reply]
Ah maybe because I only searched in this repo. Probably should have expanded to the org.
